core/src/main/java/com/netflix/conductor/core/reconciliation/WorkflowSweeper.java

```diff
@@ -46,10 +43,8 @@
     private static final String CLASS_NAME = WorkflowSweeper.class.getSimpleName();
 
     @Autowired
-    public WorkflowSweeper(WorkflowExecutor workflowExecutor,
-                           Optional<WorkflowRepairService> workflowRepairService,
-                           ConductorProperties properties,
-                           QueueDAO queueDAO) {
+    public WorkflowSweeper(WorkflowExecutor workflowExecutor, Optional<WorkflowRepairService> workflowRepairService,
```

Comment:
The formatting in many places seem off. Maybe try reformatting in the IDE before commit?

Reply:
tried re-formatting but to no effect, will fix in a later pr